upmc-enterprises/cda2fhir
# Entities stu3

#### What does this PR do?

Updates the organization name field to move any additional names present to the organization.alias field.

#### Related JIRA tickets:

[UPMCFHIR-142](https://jira.amida.com/browse/UPMCFHIR-142)

#### How should this be manually tested?

Look at the test and run jUnit. Check the output of our test data to ensure the name is populated on any organizations.

#### Background/Context:

Part of the STU3 migration.

#### New JIRA tickets for clinical required?

Nope.
